scripts/mkcloud

```diff
@@ -56,11 +56,17 @@ mkcconf=mkcloud.config
 rm -f $mkcconf # we dont want to source old info in the line below
 
 : ${virtualcloud:=virtual}
-: ${net_admin:=192.168.124}
+#: ${net_admin:=192.168.124}
+if (( $want_ipv6 > 0 )); then
+    echo ">>> WARNING: IPv6 (want_ipv6) is in development. And currently will fail to build a cloud. <<<"
```

Comment:
I think this is a little too heavy handed. "WARNING: IPv6 (want_ipv6) is under active development."